### ISSUE TITLE
elementId is wrong when I save a list

### ISSUE THREAD

**Issue by S1lentium** [barrelstrength/craft-sprout-lists#11]
When I try save a list, and reference subscriber to another userId via property elementId, I got incorrect elementId.

```php
$listType = SproutLists::$app->lists->getListType($subscriber->listType);
$lists = $listType->getLists($subscriber);
```
$lists[0]->elementID is not correct

**Comment by BenParizek** [comment]
Thanks, @S1lentium.

We'll get your pull request merged in shortly.

**Comment by BenParizek** [comment]
@S1lentium 

We believe this is resolved in Sprout Lists v1.0.0-beta.10. Let us know if you run into anything else.
